js/src/components/help-icon-button.js

```diff
@@ -16,7 +16,7 @@ import AppIconButton from '.~/components/app-icon-button';
  *
  * @param {Object} props Props
  * @param {string} props.eventContext Context to be used in `gla_help_click` track event.
- * @return {import(".~/components/app-icon-button")} The button.
+ * @return {import(".~/components/app-icon-button").default} The button.
```

Comment:
❓  What's the relation of this within the PR?

Reply:
Otherwise, it's not parsable by jsdoc. I guess it's the effect of us using "almost" ES modules ;)